### DISCUSSION TITLE
目前项目使用的版本为3.0.5-beta7，没有 downloadFullImage 这个api

### DISCUSSION THREAD

**Discussion by 52jaden** [antvis/G6#4289] [Q&A]
有没有什么办法，在不升级版本的情况下，达到 downloadFullImage 这个效果嘛
因为图形较复杂，升级版本后，导致显示完全不正确，升级成本太大

**Comment by lavenderMF** [comment]
https://github.com/antvis/G6/blob/master/packages/pc/src/graph/graph.ts#L348
